subsys/mgmt/mcumgr/grp/os_mgmt/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2018-2021 mcumgr authors
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# OS Management group public API is exposed by MCUMGR API interface,
+# when Statistics Management is enabled.
+add_library(subsys__mgmt__mcumgr__grp__os src/os_mgmt.c)
```

Comment:
`subsys__mgmt__mcumgr__grp__os` is 29 characters, this is very long, has this been tested in a very long path on windows to ensure it still builds? Can the double underscores be replaced with single underscores?

Reply:
I was also considering renaming the `subsys__mgmt_mcumgr` as MCUMGR, as it was previously.
I have taken example from another subsystems, but I am not attached to naming, so can rename it. I am open to suggestions here.